### ISSUE TITLE
Import/export track UUID

### ISSUE THREAD

**Issue by dennisguse** [OpenTracksApp/OpenTracks#307]
Import and export Track.UUID for KMZ/KML/GPX.
If a track with a specific UUID already exists in the database, do not import it again.

Some ideas how to export the UUID to the file formats can be found here: https://github.com/OpenTracksApp/OpenTracks/issues/294#issuecomment-648094869

**Comment by dennisguse** [comment]
Our KML already uses atom to add:
```
        <atom:author>
            <atom:name><![CDATA[OpenTracks]]></atom:name>
        </atom:author>
```

So, we can just use `<atom:id>` to export the UUID.

https://tools.ietf.org/html/rfc4287#page-18

**Comment by dennisguse** [comment]
UUID import/export works.
We know need to add some manual conflict resolution if a track that is already in our database is tried to re-import.
I guess we need a dialog asking if it should be replaced or not.
As the import UI was refactored, we just need to create a new DialogFragment.

**Comment by dennisguse** [comment]
Refactoring of import (regarding UI) is done and was released.

So, next thing that I found is that the `AbstractTrackImporter` supports to import multiple tracks from per file.
I am not sure if this is supported by GPX/KML or if it is a leftover from earlier (like CSV support).

After figuring this one out, I just need to show a dialog (on import conflict, i.e., same UUID in DB as to be imported) to ask the user what to do (either not import or replace the DB content).
